OCP-Infrastructure/helm/hogarama/charts/amq/Chart.yaml

```diff
@@ -0,0 +1,2 @@
+name: amq
```

Comment:
We should rethink the file structure. Why is "helm" so prominent. Do we care about technologies or services (components) we want to install? I want to install a messaging service (amq) and I don't care wether it is done with helm or OpenShift templates. That might be one reason why broker.xml is duplicated three times.